pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -82,11 +86,18 @@ func (p Provisioner) Provision(options *apibkt.BucketOptions) (*bktv1alpha1.Obje
 	if err != nil {
 		return nil, err
 	}
-	logger.Infof("Provision: creating bucket %q for OBC %q", p.bucketName, options.ObjectBucketClaim.Name)
+	logger.Infof("Provision: creating bucket %q for OBC %q for user %s", p.bucketName, options.ObjectBucketClaim.Name, options.UserID)
 
-	p.accessKeyID, p.secretAccessKey, err = p.createCephUser(options.UserID)
-	if err != nil {
-		return nil, errors.Wrap(err, "Provision: can't create ceph user")
+	if UserID(options.ObjectBucketClaim.Spec.AdditionalConfig) != "" {
```

Comment:
Please  include check in which  User is not modified by checking user details from OB, if some one tries to modify then error out.